docs/src/2_Development/02_Design_Document.md

```diff
@@ -46,3 +46,5 @@ context determines how good the outcome is.
 2. **PP = HP = AP.** During rests, players decide which of these reources to replenish.
 Characters may have different maximum, but expending one should feel roughly equivalent
 to the others.
+
+3. **Target Cards remain on a character at all times.** The Target Card represents a character moving through space, and in between combats the TC remains active on a character for any non-combat related checks. A new TC is drawn at the start of the next combat to keep cards flowing and 'shuffle' the turn order around for each combat. 
```

Comment:
1. What do you mean by 'keep the cards flowing'?
2. Should we be applying a mod to combat order that represents how quick someone is? Or "Above +2 AGL, draw combat TC at Upper Hand"?
3. Should this doc say something about why TCs need to rotate? So PCs can aim at different cards. Theoretically, a low-AR enemy could be impossible to hit if a PC already discarded all relevant cards.

```suggestion
4. **Target Cards remain on a character at all times.** Each character is always represented by a Target Card. Between combats, the TC remains active on a character for any other Checks. A new TC is drawn at the start of the next combat to shuffle the turn order around for each combat. 
```

Reply:
Added a new bullet under 3 to account for your 3rd point, and changed the language to your suggested language.

Reply:
Removed 'keep cards flowing' - original point was around encouraging more drawn cards throughout an adventure but that's less relevant to this specific design axiom and more related to the GM guide so happy to remove it here.

Reply:
I think a Power for drawing TCs with Upper Hand would solve for that rather than applying mods - the reason I like how fast drawing TCs to start combat is how clear it is what order to go in. Once we get into adding mods it slows things down and Upper Hand should be reserved for a Passive Power.